src/Resque/Job.php

```diff
@@ -330,7 +332,7 @@ public function perform()
 
             $this->complete();
 
-        // setUp said don't perform this job
+            // setUp said don't perform this job
```

Comment:
The comment should probably be aligned with the catch block, although php-cs-fixer complains... Maybe move it down?